### ISSUE TITLE
Project 2 can we use null?

### ISSUE THREAD

**Issue by ong6** [nus-cs2030/2021-s1#408]
Can we use null in project 2? Since they don't allow it in project 1.

**Comment by tanruiquan** [comment]
I think we can use null. Also, I don't recall not being able to use null in Project 1?

**Comment by zimmmz** [comment]
I think we can I did use a few null and code crunch didn't terminate the grading? but don't misuse it ah.

**Comment by descentbw** [comment]
Though CodeCrunch doesn't restrict the use of null in project 2, remember that the project is graded based on your implementation. Use null if you deem it appropriate, I guess.

**Comment by merylseow** [comment]
If you’re talking about using null to initialise variables in constructor I think it’s ok, otherwise must use with discretion I guess

**Comment by SkyBlaise99** [comment]
Using `null` to specify last `Event` rather than having a `This_Is_Last_Event extends Event`? Is this ok?

**Comment by kohwenjie** [comment]
@SkyBlaise99 i asked my TA about it for project 1 and he said its fine, so it should be fine here too i guess

**Comment by SkyBlaise99** [comment]
@kohwenjie yup asked my TA too. He say as long you can get A and pass Checkstyle shouldnt be a big issue. Thanks for helping me ask.

**Comment by kheekheekhee** [comment]
I used it and it passed the test cases so I would assume you can

**Comment by TarciusW** [comment]
yes you can!
